public/css/doctors.css

```diff
@@ -0,0 +1,100 @@
+*{
+    padding:0;
+    margin: 0;
+    font-family: lato;
+}
+main section .container{
+    width: 90%;
+    margin: 0 auto;
+    margin-top: 150px;
+}
+main section .container h1{
+    width: fit-content;
+    font-size: 40px;
+    margin: 0 auto;
+    border-bottom:1px #333 solid;
+    font-weight: 100;
+}
+main section .container .doctors{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 100%;
+    margin-top: 50px;
+}
+/******************** Doctor Style ***********************/
+main section .container .doctors .doctor{
+    width: 25%;
+    height: 600px;
+    margin-bottom: 50px;
+}
+main section .container .doctors .doctor .img{
+    width: 250px;
+    margin: 0 auto;
+    height: 250px;
+    background-image:url('https://myblue.bluecrossma.com/sites/g/files/csphws636/files/inline-images/Doctor%20Image%20Desktop.png');
+    background-position: center;
+    background-size: cover;
+    border-radius: 50%;
+}
+/******************** Brief Of Doctor ********************/
+main section .container .doctors .doctor article{
+    margin-top: 50px;
+}
+main section .container .doctors .doctor article span{
+    display: block;
+    text-align: center;
+    margin-top: 20px;
+}
+main section .container .doctors .doctor article .doctor_name{
```

Comment:
The same thing here from 49-54 it is the same property and same value 
you could've used a single class for the two referring to the previous reply